deploy/crds/quickstart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: wildfly.org/v1alpha1
+kind: WildFlyServer
+metadata:
+  name: quickstart
+spec:
+  applicationSource:
+    sourceRepository:
+      url: "https://github.com/jmesnil/wildfly-operator-quickstart.git"
```

Comment:
is using your fork here intentional for final implementation or is it only for development and it should be fixed before merging?

Reply:
fixed, the example now uses WildFly `helloworld-rs` quickstart